src/EditorFeatures/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests.cs

```diff
@@ -55,8 +55,11 @@ private static SyntaxToken GetStringToken(string text)
                     allowIndexOutOfRange, allowNullReference, allowOutOfMemory, allowDiagnosticsMismatch);
             }
 
-            var actual = TreeToText(sourceText, tree).Replace("\"", "\"\"");
-            Assert.Equal(expected.Replace("\"", "\"\""), actual);
+            const string DoubleQuoteEscaping = "\"\"";
+            var actual = TreeToText(sourceText, tree)
+                .Replace("\"", DoubleQuoteEscaping)
+                .Replace("&quot;", DoubleQuoteEscaping);
+            Assert.Equal(expected.Replace("\"", DoubleQuoteEscaping), actual);
```

Comment:
The german translation uses double quotes in the message.
https://github.com/dotnet/roslyn/blob/e48c86563b48bc290d4dd90fd348bda2ac86d915/src/Features/Core/Portable/xlf/FeaturesResources.de.xlf#L1913-L1917